content/kubermatic/main/tutorials-howtos/gitops-argocd/_index.en.md

```diff
@@ -0,0 +1,241 @@
++++
+title = "GitOps via ArgoCD"
+date = 2024-07-30T12:00:00+02:00
+weight = 56
+linktitle = "GitOps via ArgoCD"
++++
+
+{{% notice warning %}}
+This is a Alpha version of Kubermatic management via GitOps which can become the default way to manage KKP in future. This article explains how you can kickstart that journey with ArgoCD right now. But this feature can also change significantly and in backward incompatible ways. **Please use this setup in production at your own risk.**
+{{% /notice %}}
+
+
+## Need of GitOps solution
+Kubermatic Kubernetes Platform is very versatile solution to create and manage Kubernetes clusters (user-clusters) across plethora of Cloud providers and on-prem virtualizaton platforms. But this flexibility also means that there are good amount of moving parts. Kubermatic Kubernetes Platform (KKP) provides various tools to manage user-clusters across various regions and cloud.
+
+This is why, if we utilize a GitOps solution to manage KKP and it's upgrades, KKP administrator would have better peace of mind. We have now provided [alpha release of ArgoCD based management of KKP master and seeds](https://github.com/kubermatic/kubermatic/tree/main/charts/gitops/kkp-argocd-apps).
+
+This page outlines how to install ArgoCD and team provided Apps to manage KKP seeds and their upgrades.
+
+## Preparation
+
+In order to setup and manage KKP using a GitOps solution, one must first know some basics about KKP. Following links could be useful to get that knowledge, if you are new to KKP.
+
+1. [KKP main documentation home](../../) For general overview. This documentation is quite vast and you should glance through this documentation but focus on specific links below.
+1. [KKP Architecture, terminology and planning](../../architecture/)
+1. [Hardware requirements, firewall requirements](../../architecture/requirements/cluster-requirements/) and [supported cloud providers](../../architecture/supported-providers/) and [DNS requirements](../../installation/install-kkp-ce/#update-dns--tls)
+
+We will install KKP along the way, so, we do not need a running KKP installation. But if you already have KKP installation running, you can still make use of this guide to onboard your existing KKP installation to ArgoCD. This might involve in directory re-organization, though.
```

Comment:
* have _a_ KKP installation
* "This might involve in directory re-organization, though." -- not sure what this means.

Reply:
> "This might involve in directory re-organization, though."

I removed this from here.. because I have elaborated text below